### DISCUSSION TITLE
Data Format

### DISCUSSION THREAD

**Discussion by nitesh31mishra** [awslabs/gluonts#1493] [General]
Is there any compulsion that our data should have a time-frequency in minutes???
 Like I have a dataset where frequency between each data is in 35 seconds and why I pass freq parameter in DeepArEstimator as **freq = "35S"(or "35s" or "35sec")** I get an issue saying minutes should be provided......... 
 
I have tried for **freq="0.58min"** to but its showing minutes should be in integer value..

**Comment by pratikgehlott** [comment]
same issue , @mbohlkeschneider , @lostella can u guys suggest something?
